### PR TITLE
ec2_* Sanity corrections

### PR DESCRIPTION
**Pull request by jillr**
##### SUMMARY
Cleanup some typos that were missed in #64230

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/amazon/ec2_eip.py
lib/ansible/modules/cloud/amazon/ec2_launch_template.py
lib/ansible/modules/cloud/amazon/ec2_lc.py

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/ec2_launch_template.py [hunk 1]
```diff
@@ -78,7 +78,7 @@
           type. After you connect to the instance, you must mount the volume.
         type: str
       ebs:
-        description: Parameters used to automatically set up EBS volumes when the instance is launchedself
+        description: Parameters used to automatically set up EBS volumes when the instance is launched
```
Comment (felixfontein) [inline lib/ansible/modules/cloud/amazon/ec2_launch_template.py]:
```suggestion
        description: Parameters used to automatically set up EBS volumes when the instance is launched.
```

**Review by felixfontein — COMMENTED** [review]
Looks good, just found a little improvement ;)

**Review by willthames — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@jillr thanks for fixing this!
@willthames thanks for reviewing!

@jillr @tremble will someone of you backport this (together with #64230 and potentially more of these)?

**Comment by tremble** [comment]
I do intend on backporting my cleanup to 2.9. 

1. I need to make sure I understand the process (especially since I included a bunch of deprecations)
2. I need some clear time again.  (I wanted to get everything through as a higher priority just so it's "done")

**Comment by felixfontein** [comment]
The general backporting process is described here: https://docs.ansible.com/ansible/latest/community/development_process.html#backporting-merged-prs

Deprecations should not be backported. Simply remove that part of the code / docs / changelog fragment changes. (You can leave "the option has no effect" in there, just remove the "deprecated" / "removed in version xxx" parts.)

**Pull request merged**